sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/MessageUtils.java

```diff
@@ -60,7 +60,7 @@ private MessageUtils() {
 
     // Pass little less than client timeout to the server so client doesn't time out before server times out
     public static Duration adjustServerTimeout(Duration clientTimeout) {
-        return clientTimeout.minusMillis(1000);
+        return clientTimeout.minusMillis(5000);
```

Comment:
Why was this updated?

Reply:
will remove it, it was left from testing.